### ISSUE TITLE
Prettier unexpectedly moves docblock

### ISSUE THREAD

**Issue by leighman** [prettier/plugin-php#1546]
**@prettier/plugin-php v0.14.3**
[Playground link](https://loilo.github.io/prettier-php-playground/#N4IgDgTgpgLjCWUIHkwIPYDsDOIBcok8mMA6vACYwAW+AHAAwA0IMAhgEblW14AsLAK7YoAFU648AMzYAbES2zEA5rKgBFQehhR8M+VBZhqYAGpIlWfCACsAOj4gWMCG3iyVAYXQBbH2wAFAAkA-BdBQxAOVwBjKABlGABPNWswbAgAWgAmJxBoAEdBeGgA12V-PTkFEGIRCBgytgq2KoMAXxYoCngYdBQ0eCxJUDZsGDaRTtrMMEEJvBAAHgB+YzAAHUwtgBIfJIB9CjYdA4QfKAACAF5LzCgAd0uAegAqV8uAAXS5H0z4PzzThqS6vZ6XGKyMbYS5QAAeOkwFBhABETmIAVAAJKA9gcEHAdoAbhA7SAA)

**Input:**
```php
<?php

$my_date_time = new /** @psalm-immutable */ class extends DateTimeImmutable {};
```

**Output:**
```php
<?php

$my_date_time = new class/** @psalm-immutable */  extends DateTimeImmutable {};

```

I would expect the docblock to annotate the class and not be moved.

The output is not valid with psalm https://psalm.dev/r/bf04faa3c1

**Comment by czosel** [comment]
Indeed, thanks for the bug report @leighman!

**Comment by czosel** [comment]
Released as `v0.16.0` :tada:
